openedx/core/djangoapps/discussions/docs/decisions/0004-transition-legacy-experience.rst

```diff
@@ -0,0 +1,56 @@
+Transition away from legacy discussions experience
+==================================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+To date, all courses in the LMS are automatically provided with
+Discussions via the legacy EdX system (`cs_comments_service`).
+The Discussions Tab cannot be hidden via Studio (though possibly via
+manual XML edits during Course Import?).
+
+
+Requirements
+------------
+
+- The existing discussions experience should be configurable via the new
+  Course Authoring MFE _before_ we switch to the new Discussions MFE
+  experience.
+- Existing courses should not be prompted to select a discussion
+  provider; they get the legacy experience, unless manually overridden
+  via Djano Admin.
+- Only new courses, after broad rollout, should be prompted to select an
+  available provider.
```

Comment:
One note here - only new courses after a certain date will get the new discussions experience by default upon course creation, but we want to show new providers whenever v1.5 rolls out, which is before  milestones for the new discussions experience.

Reply:
To me this means there's probably a task in v1.5 where we switch from configuring legacy discussions to auto-configuring the "new" discussions... which should look 95% the same, I expect, in terms of where data is stored.  It may just mean we start auto-configuring a discussion configuration for the "new" discussions instead of the legacy ones.